docs/docker/docker_demo/fast_dds_suite.rst

```diff
@@ -0,0 +1,137 @@
+.. _fast_dds_suite:
+
+Fast DDS Suite
+==============
+
+This Docker image contains the complete Fast DDS suite. This includes:
+
+- :ref:`eProsima Fast DDS libraries and examples <fast_dds_suite_examples>`: Fast DDS libraries bundled with several
+  examples that showcase a variety of capabilities of eProsima's Fast DDS implementation.
+
+- :ref:`Shapes Demo <fast_dds_suite_shapes_demo>`: eProsima Shapes Demo is an application in which Publishers and
+  Subscribers are shapes of different colors and sizes moving on a board. Each shape refers to its own topic: Square,
+  Triangle or Circle. A single instance of the eProsima Shapes Demo can publish on or subscribe to several topics at
+  a time.
+
+  You can read more about this application on the `Shapes Demo documentation page <https://eprosima-shapes-demo.readthedocs.io/>`_.
```

Comment:
I would remove this line and instead add a link to Fast DDS documentation in the previous element.